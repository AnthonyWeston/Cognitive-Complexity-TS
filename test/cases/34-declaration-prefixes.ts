export function exportDeclaration() {
    while (true);
}

export const exportAnonymous = () => {
    while (true);
};
