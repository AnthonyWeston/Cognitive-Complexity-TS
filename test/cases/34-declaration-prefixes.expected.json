{
    "34-declaration-prefixes.ts": {
        "score": 2,
        "inner": [
            {
                "name": "exportDeclaration",
                "score": 1,
                "line": 1,
                "column": 1
            },
            {
                "name": "exportAnonymous",
                "score": 1,
                "line": 5,
                "column": 32
            }
        ]
    }
}
